bin.src/transform_all_shards.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+from lsst.the.monster import GaiaDR3Info, MatchAndTransform
+import glob
+
+
+GaiaDR3CatInfoClass = GaiaDR3Info
+gaia_info = GaiaDR3CatInfoClass()
+
+# Make a list of all the Gaia shards:
+fits_list = glob.glob(gaia_info.path+'/*.fits')
```

Comment:
This should probably be `sorted(glob.glob())` to make sure it's repeatable from run to run.